app/test/obituary_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'bundler'
+Bundler.require :default, :test
+require_relative '../lib/obituary_microservice.rb'
+
+class ObituaryTest < Minitest::Test
+	include Rack::Test::Methods
+
+	def app
+    ObituaryMicroservice.new
+	end
+
+  def test_it_returns_a_200
+  	get '/'
+  	assert last_response.ok?
+  end
+
+  def test_it_returns_formatted_obituaries_json
+  	get '/'
+  	assert_equal 'obituary', JSON.parse(last_response.body).first['data']['type']
+  end
```

Comment:
Definitely need to add more tests once we know what we want to do with this

Reply:
Way to start writing test!